### DISCUSSION TITLE
Typo in MacOS compilation instruactions

### DISCUSSION THREAD

**Discussion by CpHaddock** [sebalamm/KaMIS#17] [General]
It seems that the path separator in `HowToMacOs.md` file is incorrect, it should be a semicolon instead of a colon:

It is: `export PATH=/usr/local/bin:$PATH`
Should be: `export PATH=/usr/local/bin;$PATH`

**Comment by schulzchristian** [comment]
Hi, no the : is correct. ; stops setting the path and you would loose the remaining path.

Best
Christian

**Reply by CpHaddock** [reply]
You're right! My bad! I was setting paths from the command line and not bashrc.
